lib/fauxpaas/cli.rb

```diff
@@ -46,7 +46,7 @@ def initialize
       command :deploy do |c|
         c.action do |global_options, _options, args|
           invoker.add_command(
-            DeployCommand.new(
+            Deploy.new(
```

Comment:
I'm not actually sure why this can be resolved as a bare `Deploy` and doesn't require `Commands::Deploy`. Is there something in GLI that imports symbols from a commands module?